subprojects/launcher/src/main/java/org/gradle/tooling/internal/provider/TestExecutionRequestAction.java

```diff
@@ -53,7 +58,17 @@ public static TestExecutionRequestAction create(BuildClientSubscriptions clientS
         return new TestExecutionRequestAction(clientSubscriptions, startParameter,
                                                 ImmutableSet.copyOf(testExecutionRequest.getTestExecutionDescriptors()),
                                                 ImmutableSet.copyOf(testClassNames),
-                                                providerInternalJvmTestRequests);
+                                                providerInternalJvmTestRequests,
+                                                getDebugOptions(testExecutionRequest));
+    }
+
+    private static InternalDebugOptions getDebugOptions(ProviderInternalTestExecutionRequest testExecutionRequest) {
```

Comment:
What's the better way to handle this? Is there a way to adapt older TAPI client requests to use the default value?

Reply:
Not that I know of. @oehme already approved this approach.

Reply:
I will resolve this conversation since it was already approved.